plugins/action/k8s_info.py

```diff
@@ -92,22 +92,38 @@ def load_template(self, template, new_module_args, task_vars):
                     if value is not None and not isinstance(value, string_types):
                         raise AnsibleActionFail("%s is expected to be a string, but got %s instead" % (s_type, type(value)))
             try:
-                trim_blocks = boolean(template_args.get('trim_blocks', True), strict=False)
-                lstrip_blocks = boolean(template_args.get('lstrip_blocks', False), strict=False)
+                template_param.update(dict(trim_blocks=boolean(template_args.get('trim_blocks', True), strict=False)))
+                template_param.update(dict(lstrip_blocks=boolean(template_args.get('lstrip_blocks', False), strict=False)))
             except TypeError as e:
                 raise AnsibleActionFail(to_native(e))
 
-            newline_sequence = template_args.get('newline_sequence', self.DEFAULT_NEWLINE_SEQUENCE)
-            variable_start_string = template_args.get('variable_start_string', None)
-            variable_end_string = template_args.get('variable_end_string', None)
-            block_start_string = template_args.get('block_start_string', None)
-            block_end_string = template_args.get('block_end_string', None)
+            template_param.update(dict(newline_sequence=template_args.get('newline_sequence', self.DEFAULT_NEWLINE_SEQUENCE)))
```

Comment:
Two thing here:
- Can you please use the more Pythonic `{}` instead of `dict()`.
- You can do on single update() if you pass a multi keys `dict. update({"newline_sequence": blabla, "variable_start_string": blabla, "variable_end_string": blabla})`.